src/status_im/activity_center/core.cljs

```diff
@@ -1,6 +1,6 @@
 (ns status-im.activity-center.core
-  (:require [re-frame.core :as re-frame]
-            [status-im.constants :as constants]
+  (:require [re-frame.core :as rf]
+            [status-im.constants :as c]
```

Comment:
No strong opinion tbh, but I'd prefer more descriptive aliases over shorter (if cannot do both). Maybe use `const` to make it shorter as it is currently but keeping the self-descriptiveness?

Reply:
Done :)